### PR TITLE
add contributors for ambient

### PR DESCRIPTION
**Pull request by linsun**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [ x ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: .spelling [hunk 1]
```diff
@@ -26,6 +26,7 @@
 1.8.x
 1.8.x.
 1.9.x
+1.10.x
```
Comment (ericvn) [inline .spelling:29]: This is a double entry (down two lines).

File: .spelling [hunk 2]
```diff
@@ -415,6 +420,9 @@ git
 GitHub
 GitOps
 GKE
+GKE-IAM
```
Comment (ericvn) [inline .spelling:423]: I expect some of these entries may have crept in from other PRs.

File: content/en/blog/2022/introducing-ambient-mesh/index.md [hunk 3]
```diff
@@ -137,3 +137,5 @@ We would love your feedback to help shape the solution.
 A build of Istio which supports ambient mesh is available to [download and try](/blog/2022/get-started-ambient/) in the [Istio Experimental repo]({{< github_raw >}}/tree/experimental-ambient).
 A list of missing features and work items is available in the [README]({{< github_raw >}}/blob/experimental-ambient/README.md).
 Please try it out and let us know what you think!
+
+*Contributors: Aaron Birkland (Solo.io), Craig Box (Google), Greg Hanson (Solo.io), Daniel Hawton (Solo.io), John Howard (Google), Ethan J. Jackson (Google), Denis Jannot (Solo.io), Abhi Joglekar (Google), Yuval Kohavi (Solo.io), Steven Landow (Google), Idit Levine (Solo.io), Oliver Liu (Google), Yossi Mesika (Solo.io), Justin Pettit (Google), Neeraj Poddar (Solo.io), Nina Polshakova (Solo.io), Christian Posta (Solo.io), Doug Reid (Google), Louis Ryan (Google), Lin Sun (Solo.io), Eitan Yarmush (Solo.io), Kuat Yessenov (Google), Francis Zhou (Google)*
```
Comment (craigbox) [inline content/en/blog/2022/introducing-ambient-mesh/index.md:141]:
```suggestion
*Thank you to the team that contributed to the launch of ambient mesh: Aaron Birkland, Craig Box, Greg Hanson, Daniel Hawton, John Howard, Ethan J. Jackson, Denis Jannot, Abhi Joglekar, Yuval Kohavi, Steven Landow, Idit Levine, Oliver Liu, Yossi Mesika, Justin Pettit, Neeraj Poddar, Nina Polshakova, Christian Posta, Doug Reid, Louis Ryan, Lin Sun, Eitan Yarmush, Kuat Yessenov and Francis Zhou*
```

Comment (craigbox) [reply content/en/blog/2022/introducing-ambient-mesh/index.md:141]: That list is an eyesore with the affiliations.  With so many names, it's also better for the community to adjust it in the proposed fashion.

Comment (linsun) [reply content/en/blog/2022/introducing-ambient-mesh/index.md:141]: I can take your other suggestions but i think we have agreed to put company names.

File: .spelling (hunk 2 above)
Comment (linsun) [reply .spelling:421]: yep

File: .spelling (hunk 1 above)
Comment (linsun) [reply .spelling:29]: thanks, fixed

File: content/en/blog/2022/introducing-ambient-mesh/index.md [hunk 4]
```diff
@@ -137,3 +137,5 @@ We would love your feedback to help shape the solution.
 A build of Istio which supports ambient mesh is available to [download and try](/blog/2022/get-started-ambient/) in the [Istio Experimental repo]({{< github_raw >}}/tree/experimental-ambient).
 A list of missing features and work items is available in the [README]({{< github_raw >}}/blob/experimental-ambient/README.md).
 Please try it out and let us know what you think!
+
+*Thank you to the team that contributed to the launch of ambient mesh: Aaron Birkland (Solo.io), Craig Box (Google), Greg Hanson (Solo.io), Daniel Hawton (Solo.io), John Howard (Google), Ethan J. Jackson (Google), Denis Jannot (Solo.io), Abhi Joglekar (Google), Yuval Kohavi (Solo.io), Steven Landow (Google), Idit Levine (Solo.io), Oliver Liu (Google), Yossi Mesika (Solo.io), Justin Pettit (Google), Neeraj Poddar (Solo.io), Nina Polshakova (Solo.io), Christian Posta (Solo.io), Doug Reid (Google), Louis Ryan (Google), Lin Sun (Solo.io), Eitan Yarmush (Solo.io), Kuat Yessenov (Google) and Francis Zhou (Google)*
```
Comment (craigbox) [inline content/en/blog/2022/introducing-ambient-mesh/index.md:141]:
```suggestion
*Thank you to the team that contributed to the launch of ambient mesh! 
* Google: Craig Box, John Howard, Ethan J. Jackson, Abhi Joglekar, Steven Landow, Oliver Liu, Justin Pettit, Doug Reid, Louis Ryan, Kuat Yessenov, Francis Zhou
* Solo.io: Aaron Birkland, Greg Hanson, Daniel Hawton, Denis Jannot, Yuval Kohavi, Idit Levine, Yossi Mesika, Neeraj Poddar, Nina Polshakova, Christian Posta, Lin Sun, Eitan Yarmush
```

Comment (linsun) [reply content/en/blog/2022/introducing-ambient-mesh/index.md:141]: @justinpettit was very much against of listing under two companies which was the initial idea.

Comment (justinpettit) [reply content/en/blog/2022/introducing-ambient-mesh/index.md:141]: I was, but there are now a lot more contributors listed than I thought we'd have.  If we're not going to trim the list, I'd prefer Craig's suggestion.

Comment (linsun) [reply content/en/blog/2022/introducing-ambient-mesh/index.md:141]: ok cool! that is fine with me too!  Thanks to both!!

Comment (ericvn) [inline content/en/blog/2022/introducing-ambient-mesh/index.md]: As an FYI, I broke this lint into multiple lines to get past the lint failure and it still showed as italics:
```suggestion
*Thank you to the team that contributed to the launch of ambient mesh: Aaron Birkland (Solo.io), Craig Box (Google),
Greg Hanson (Solo.io), Daniel Hawton (Solo.io), John Howard (Google), Ethan J. Jackson (Google), Denis Jannot (Solo.io),
Abhi Joglekar (Google), Yuval Kohavi (Solo.io), Steven Landow (Google), Idit Levine (Solo.io), Oliver Liu (Google),
Yossi Mesika (Solo.io), Justin Pettit (Google), Neeraj Poddar (Solo.io), Nina Polshakova (Solo.io),
Christian Posta (Solo.io), Doug Reid (Google), Louis Ryan (Google), Lin Sun (Solo.io), Eitan Yarmush (Solo.io),
Kuat Yessenov (Google) and Francis Zhou (Google)*
```

Comment (linsun) [reply content/en/blog/2022/introducing-ambient-mesh/index.md]: Yes, that is great to know! Good way to get the lint fixed.

**Review by ericvn — APPROVED** [review]

**Comment by ericvn** [comment]
Thanks Lin for working through the lint issues on this one.

**Comment by linsun** [comment]
Thank you Eric, much appreciate your help! and @craigbox's too!

File: .spelling [hunk 5]
```diff
@@ -1042,14 +1053,24 @@ Xeon
 Yahya
 yaml
 yamls
+Yarmush
+Yessenov
+yournamespace
+Youtube
```
Comment (craigbox) [inline .spelling:1059]: these two don't look like they belong here

File: .spelling [hunk 6]
```diff
@@ -1042,14 +1053,24 @@ Xeon
...
+Yossi
 Zack
 Zehuan
 Zhang
 Zhao
+Zhihan
+Zhong
+Zhonghu
+Zhou
 zhaohuabing
```
Comment (craigbox) [inline .spelling:1074]: some dupes/out of order?

File: .spelling [hunk 7]
```diff
@@ -1034,6 +1044,7 @@ x509
 x86
 xDS
 Xia
+Xining
```
Comment (craigbox) [inline .spelling:1047]: dupe

File: .spelling [hunk 8]
```diff
@@ -605,6 +612,7 @@ meetups
 memcached
 memcached-2's
 Mengxue
+Mesika
```
Comment (craigbox) [inline .spelling:615]: not sorted

File: .spelling [hunk 9]
```diff
@@ -275,6 +276,7 @@ CVE-2021-29258
 CVE-2021-29492
 CVE-2021-31920
 CVE-2021-31921
+CVE-2021-34824
```
Comment (craigbox) [inline .spelling:279]: this right?

**Review by craigbox — DISMISSED** [review]

**Comment by linsun** [comment]
@craigbox i'll submit another PR to fix spelling, for now, i want to get the contributors on ambient blog asap.

**Comment by craigbox** [comment]
(np, just thought that might stop lint succeeding?)

**Pull request merged**